components/Icons/Chevron.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const getStyle = direction =>{
```

Comment:
It seems you've not enabled prettier on your editor. Could you please prettify the files? Let me know if you need help. Thank you so much!